src/settings.rs

```diff
@@ -0,0 +1,27 @@
+#[derive(Debug)]
+pub struct Settings {
+    pub venv_from_stdlib: bool,
+    pub venv_outside_project: bool,
+}
+
+impl Default for Settings {
+    fn default() -> Settings {
+        Settings {
+            venv_from_stdlib: true,
+            venv_outside_project: false,
+        }
+    }
+}
+
+impl Settings {
+    pub fn from_env() -> Settings {
+        let mut res = Settings::default();
```

Comment:
Nit: `Default::default` is more idiomatic, I almost never see `MyStruct::default`